saskatoon/sitebase/views.py

```diff
@@ -29,6 +33,34 @@ def get_context_data(self, **kwargs):
 
         return context
 
+    def dispatch(self, request, *args, **kwargs):
+        """Redirect new pickleaders"""
+
+        user = self.request.user
+
+        # Start onboarding flow for newly onboarded pickleaders
+        if user.is_authenticated and user.roles == [dict(AUTH_GROUPS)['volunteer']]:
```

Comment:
```suggestion
        if user.is_authenticated and user.role_groups == ['volunteer']:
```

Reply:
actually, we have to be more careful here, it is possible that a property owner is attending a pickleader training,
so we should essentially check `'volunteer' in user.role_groups and 'pickleader' not in user.role_groups`

Reply:
got it. `role_groups` returns a `QuerySet` of `Group`s so i'm mapping the names into a list for legibility